src/NServiceBus.Core/Routing/HybridSubscriptions.cs

```diff
@@ -59,6 +67,12 @@ protected internal override void Setup(FeatureConfigurationContext context)
             }
         }
 
+        public static bool IsMigrationModeEnabled(ReadOnlySettings settings)
+        {
+            // this key can be set by transports once they provide native support for pub/sub.
+            return settings.TryGet("NServiceBus.Subscriptions.EnableMigrationMode", out bool enabled) && enabled;
```

Comment:
no strong opinion but I'd vote to stick with "Hybrid mode" on the core level and use "Migration" on the specific transport package using this capability.

Reply:
As mentioned on slack, I think there's a good argument for moving to "migration mode"